service/service-requirements.txt

```diff
@@ -6,4 +6,5 @@ uvicorn==0.20.0
 asgi-correlation-id==3.2.1
 starlette_prometheus==0.9.0
 singleton-decorator==1.0.0
-fastapi_utils==0.2.1
\ No newline at end of file
+fastapi_utils==0.2.1
+idna==2.10
```

Comment:
fastapi_utils automatically includes idna as a dependency during installation. If you need to specify a particular version of idna, it should be done prior to installing fastapi_utils.